.bazelrc

```diff
@@ -42,4 +42,6 @@ build:release-ios --config=release-common
 build:release-ios --copt=-fembed-bitcode
 
 # Flags for release builds targeting Android or the JVM
+build:release-android --config=android
 build:release-android --config=release-common
+build:release-android --fat_apk_cpu=x86,armeabi-v7a,arm64-v8a
```

Comment:
I think we should omit architectures here and specify them where we do individual builds, for parity with iOS. Thoughts?

Reply:
+1 to this

Reply:
done